### ISSUE TITLE
Can GCM be installed on ARM64 VM with .NET SDK 8.0?

### ISSUE THREAD

**Issue by wangglei** [git-ecosystem/git-credential-manager#1681]
From the installation instructions, the only option to install GCM on ARM64 Linux is to use version 7.0 of the .NET SDK. But this version of .NET has reached EndOfLife. Is there new installation instructions for using .NET SDK 8.0? Thanks!

**Comment by craxal** [comment]
Agreed, the required version should be updated. The only option I see is to install the .NET 7.0 runtime snap, and even then, GCM couldn't detect it.

**Comment by NVolcz** [comment]
It seems that GCM is using dotnot 8.0 for Linux, since the 2.5 release:
https://github.com/git-ecosystem/git-credential-manager/pull/1587
@wangglei, what method did you use for the installation? I am trying to figure this out for myself

**Comment by xtqqczze** [comment]
> We are currently facing some issues publishing a correctly signed package to Nuget.org. Due to recent sponsorship/corporate funding priority changes, work on GCM has unfortunately slowed significantly.
> 
> Please watch this space!

_Originally posted by @mjcheetham in https://github.com/git-ecosystem/git-credential-manager/issues/1606#issuecomment-2150748156_

**Comment by wangglei** [comment]
@NVolcz I was following the ".NET tool" section from install.md file.

**Comment by xtqqczze** [comment]
> @NVolcz I was following the ".NET tool" section from install.md file.

In that case, this issue is a duplicate of https://github.com/git-ecosystem/git-credential-manager/issues/1606.

**Comment by xtqqczze** [comment]
> GCM 2.6 which targets .NET 8 has just been released on nuget.org
> 
> https://www.nuget.org/packages/git-credential-manager/2.6.0

_Originally posted by @mjcheetham in https://github.com/git-ecosystem/git-credential-manager/issues/1702#issuecomment-2383279743_

**Comment by paulgmiller** [comment]
I get this when I use curl -L https://aka.ms/gcm/linux-install-source.sh | sh on arm64 azure vm

git-credential-manager configure
-bash: /usr/local/bin/git-credential-manager: cannot execute binary file: Exec format error

**Comment by xtqqczze** [comment]
> arm64 azure vm

@paulgmiller Which instance series and VM image are you using?

**Comment by ridgunn** [comment]
@wangglei  I'm closing this issue since .NET 7 has been updated to 8 with

@paulgmiller are you still experiencing the same issue? Could you provide more information about your Linux distribution? Please feel free to create new issue.

Additionally, arm64 package support for Linux has been completed with #2232, so I am closing this issue.
